debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerConnectorConfig.java

```diff
@@ -231,13 +231,27 @@ public static SnapshotIsolationMode parse(String value, String defaultValue) {
                     + "When '" + SnapshotIsolationMode.SNAPSHOT.getValue() + "' is specified, connector runs the initial snapshot in SNAPSHOT isolation level, which guarantees snapshot consistency. In addition, neither table nor row-level locks are held. "
                     + "In '" + SnapshotIsolationMode.READ_UNCOMMITTED.getValue() + "' mode neither table nor row-level locks are acquired, but connector does not guarantee snapshot consistency.");
 
+    public static final Field ROWS_FETCH_SIZE = RelationalDatabaseConnectorConfig.ROWS_FETCH_SIZE.withDefault(DEFAULT_ROWS_FETCH_SIZE);
```

Comment:
What's the reasoning for (re-)defining these fields here? They should be in the base class.

Reply:
Postgres has the default rows fetch set to 10240 whereas sqlserver has the default was set to 2000. The base will have one of the default set, how do we override the default in the specific database config?

Reply:
I think we can converge on default value; just go for the larger one globally.